docusaurus/docs/recipes/contentful.md

```diff
@@ -174,13 +182,12 @@ Given the nature of Next.js and the implementation of [next-redux-wrapper](https
 In the following example we will be checking the presence of a `cms-preview` query parameter to decide whether to show preview content or the standard, published content, i.e. something like `www.example.com/?cms-preview`. 
 
 ```js
-// buildStore.js
-import { createClient } form 'contentful';
+// www/shared/utils/build-store/buildStore.js
```

Comment:
I'm not sure if this is still the right path based on our redux recipe: https://next-with-moxy-recipes.moxy.tech/docs/recipes/setting-up-redux